Makefile

```diff
@@ -75,21 +70,30 @@ SIMPLE = newsflash.inc \
 	 source/old/index.html
 SRCLISTS = $(foreach S,$(FUTURESERIES) $(SERIES) $(OLDSERIES2) fips,source/old/$(S)/index.inc source/old/$(S)/index.html)
 
+SIMPLEDOCS = docs/faq.inc docs/fips.inc \
+	     docs/OpenSSLStrategicArchitecture.html \
+	     docs/OpenSSL300Design.html \
+	     docs/manpages.html \
+	     docs/mansidebar.html
 
 .SUFFIXES: .md .html
 
 .md.html:
 	@rm -f $@
 	./bin/md-to-html5 $<
 
-all: suball manmaster mancross sitemap akamai-purge
+all: suball docs sitemap akamai-purge
```

Comment:
This seems to actually change the "all" target. Previously only `manmaster` and `mancross` was included. Now its `manpages` and `mancross`. So `manpages` instead of `manmaster` which seems to encompass more. Is that change intentional?

Reply:
Fixed